src/modules/ejector/ejector.py

```diff
@@ -105,6 +106,19 @@ def build_report(self, blockstamp: ReferenceBlockStamp) -> tuple:
 
         EJECTOR_VALIDATORS_COUNT_TO_EJECT.set(report_data.requests_count)
 
+        try:
+            with open('validators_response.json', 'w') as f:
+                f.write(
+                    json.dumps([{
+                        **v.__dict__,
+                        'validator': v.validator.__dict__,
```

Comment:
it works like a charm including nested dataclasses too
```
from dataclasses import asdict 

asdict(v)
```

Reply:
lol, it's very slow.
My method works 0.6107 seconds
asdict works 9.8773 seconds
with same input data (500 MB of validators)